src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinFeature.kt

```diff
@@ -58,7 +58,14 @@ enum class KotlinFeature(private val enabledByDefault: Boolean) {
      * In addition, the adjustment of behavior using get:JvmName is disabled.
      * Note also that this feature does not apply to setters.
      */
-    KotlinPropertyNameAsImplicitName(enabledByDefault = false);
+    KotlinPropertyNameAsImplicitName(enabledByDefault = false),
+
+    /**
+     * This feature represents whether to handle [kotlin.time.Duration] using [java.time.Duration] as conversion bridge.
+     *
+     * This allows use Kotlin Duration type with [com.fasterxml.jackson.datatype.jsr310.JavaTimeModule].
```

Comment:
Could you describe that annotations for `JavaTime` may not work for deserialization?

Reply:
Added additional note, not sure if that was what you meant so please recheck.

Reply:
Perhaps annotating `JsonFormat` during deserialization does not work in the current situation?
I would have liked to see a comment mentioning such incompatibility.

Reply:
My understanding of Jackson is that `@JsonFormat` is using only for serialization and doesn't affect deserialization. We have to set it either on field using `@field:JsonFormat` or getter using `@get:JsonFormat` but in `data class` it works same way as for `Instant` so nothing really changes.

Anyway, I added another test involving mixin (to check that functionality as well) just to be safe.

Reply:
> My understanding of Jackson is that @JsonFormat is using only for serialization and doesn't affect deserialization.

No. Deserialization is also affected.
See the example below.

```kotlin
data class Dto(
    @JsonFormat(pattern = "HOURS") val formatted: JavaDuration,
    val default: JavaDuration
)

@Test
fun test() {
    val mapper = jacksonObjectMapper().registerModule(JavaTimeModule())

    val json = """{"formatted":1,"default":1}"""
    val actual = mapper.readValue<Dto>(json)

    assertEquals(JavaDuration.ofHours(1), actual.formatted)
    assertEquals(JavaDuration.ofHours(1), actual.default) // -> fails, Actual: PT1S
}
```

Reply:
Added test for this case. For me behavior looks correct, default is serialized using seconds, annotation changes it to hours.